sematic/ui/packages/common/src/component/TagsList.tsx

```diff
@@ -21,13 +22,15 @@ const StyledBox = styled(Box)`
 
 interface TagsListProps {
     tags: string[];
+    onClick?: (tag: string) => void;
+    onAddTag?: () => void;
 }
 
 const TagsList = (props: TagsListProps) => {
-    const { tags = [] } = props;
+    const { tags = [], onClick, onAddTag } = props;
     return <StyledBox>
-        {tags.map(tag => <Chip key={tag} label={tag} />)}
```

Comment:
When there will be more tags than can hold on one line, we will display them on multiple lines. I tested in storybook by reducing the width of the 
component to e.g. 100px, and some spacing is missing between lines. Maybe you can simply add a bottom margin?
<img width="174" alt="Screenshot 2023-03-30 at 10 08 55 PM" src="https://user-images.githubusercontent.com/429433/228952642-3ab0b813-3d8f-4d13-ab32-f09bb9d102e6.png">

Reply:
Very nice catch! Thanks. Fixed. please check https://story.dev-usw2-sematic0.sematic.cloud/?path=/story/sematic-tags--tags-list&args=width:128